psk/server-psk-nonblocking.c

```diff
@@ -268,17 +159,153 @@ int main()
                 return 1;
             }
             wolfSSL_set_fd(ssl, connfd);
-
+            sockfd = wolfSSL_get_fd(ssl);
+            
             /* set wolfSSL and socket to non blocking and respond */
             wolfSSL_set_using_nonblock(ssl, 1);
             if (fcntl(connfd, F_SETFL, O_NONBLOCK) < 0) {
                 printf("Fatal error : fcntl set failed\n");
                 return 1;
             }
-            if (Respond(ssl) != 0) {
-                printf("Fatal error : respond error\n");
-                return 1;
+            
+            ret = wolfSSL_accept(ssl);
+            error = wolfSSL_get_error(ssl, 0);
+            
+    /* clearing buffer for client reponse to prevent unexpected output*/
```

Comment:
Indentation looks like it's off here, is this inside the above else block?  If so, please adjust indentation of this and code following it.